tests/Storage.test.ts

```diff
@@ -292,43 +292,61 @@ var sample_data =
   ];
 
 /**
- * Creates createLedgerStorage
+ * Run LedgerStorageTest
  */
-function createLedgerStorage ()
+function runLedgerStorageTest ()
 {
-    var ledger_storage: LedgerStorage = new LedgerStorage(":memory:", (err1: any) =>
+    var ledger_storage: LedgerStorage = new LedgerStorage(":memory:", (err1: Error | null) =>
     {
-        assert.ok(!err1, err1);
+        assert.ok(!err1, err1?.message);
 
-        putAllBlockData(ledger_storage, (err2: any) =>
+        runBlockTest(ledger_storage, () =>
         {
-            assert.ok(!err2, err2);
-
-            getBlockData(ledger_storage, 1, (err3: any) =>
+            runTransactionsTest(ledger_storage, () =>
             {
-                assert.ok(!err3, err3);
+                runEnrollmentsTest(ledger_storage, () =>
+                {
+                    ledger_storage.close();
+                });
+            });
+        });
+    });
+}
+
+/**
+ * Puts all data
+ */
+function runBlockTest (ledger_storage: LedgerStorage, callback: () => void)
+{
+    putAllBlockData(ledger_storage, (err2: Error | null) =>
+    {
+        assert.ok(!err2, err2?.message);
+
+        getBlockData(ledger_storage, 1, (err3: Error | null) =>
+        {
+            assert.ok(!err3, err3?.message);
 
-                ledger_storage.close();
-              });
+            callback();
```

Comment:
The test code for putAllBlockData is written.
However, it is not actually connected to the Stoa API.
https://github.com/bpfkorea/stoa/blob/5b3252c8abb9a215ea7c4b8903fc8cb2111a2269/src/Stoa.ts#L59
You need to change the code of this Stoa to associate block data storage with the API.

Reply:
Good Point

Reply:
I connected another methods to the end of LedgerStorage.putBlocks